gradio/components/bar_plot.py

```diff
@@ -126,6 +126,8 @@ def __init__(
         self.height = height
```

Comment:
Lets add to the docstring for `width` that if no `width` is passed, then it takes up the full width of the container.

Reply:
What happens if no `height` is passed? I'm actually not sure